ui/partials/main/thInfraMenu.html

```diff
@@ -15,10 +15,6 @@
   <a class="dropdown-item" href="https://secure.pub.build.mozilla.org/builddata/reports/slave_health/"
      target="_blank" rel="noopener">Slave Health</a>
 </li>
-<li>
-  <a class="dropdown-item" href="https://api.pub.build.mozilla.org/clobberer/"
```

Comment:
Hi! Is there a taskcluster equivalent to clobberer, that we need to link to instead?

Reply:
@edmorley not really in a way clobberer was. There is a way to clear cache on each task. that is why I chose to just remove the link.